lib/season_stats_JP.rb

```diff
@@ -0,0 +1,105 @@
+module SeasonStatsJP
+
+  def total_games(data = nil)
+    data = @data if data == nil
+    data.uniq do |hash|
+      hash[:game_id]
+    end.length
+  end
+
+  def total_wins(data = nil)
```

Comment:
I think that total_wins can be done by calling total_games with a subset of data where `{won: true}`

Reply:
Great point, this has been updated!